### ISSUE TITLE
A11y_Azure Data Studio_Nov2023_Database Properties Preview_Filegroups_ScreenReader: NVDA is not announcing anything about check box when we navigate with arrow key in database table.

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/azuredatastudio#25131]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
## Git hub Tags:
#A11yTCS; #A11yMAS; #A11ySev3; #DesktopApp; #WCAG1.3.1; #FTP; #SH_Azure Data Studio_Nov2023_Win32_Dec2023; #SQL Azure Data Studio; #Win32; #NVDA; #Element:Checkbox #NonBenchmark;
## Environment Details:
Application Name: Azure Data Studio
Operating System: Win11
Azure data studio 1.47.0
Jaws [2023.2212.23]
NVDA [2023.3.0]
## Repro Steps:
1. Turn on NVDA screen reader
2. Install Azure data studio application and open it
3. Make sure to enable "Preview" feature in settings.
4. Tab till "Database" and press enter to expand the tree.
5. Tab till any database and pressF10 to open context menu.
6. Tab till "Properties Preview" and press enter.
7. Tab till "Filegroup" tab and press enter.
8. Tab till "Database files" table and navigate with arrow key in second row.
9.  You will observe that NVDA is not announcing anything about check box when we navigate with arrow key in database table.
## Actual Result:
NVDA is not announcing anything about check box when we navigate with arrow key in database table.
Note: Issue seen throughout the application for similar type control and issue is not seen with JAWS screen reader. 
## Expected Result:
NVDA should announce about check box when we navigate with arrow key in database table.
## User Impact:
Screen reader user will not get the information about checkbox. 
### Attachment:
NVDA Attachment
https://github.com/microsoft/azuredatastudio/assets/124254981/ec5e28dd-d1d6-4243-a69a-38e8768bbd51
Jaws Attachment
https://github.com/microsoft/azuredatastudio/assets/124254981/c712243c-6b70-4b49-acd2-675e82f37847
Code Snippet
![25131_Code Snippet](https://github.com/microsoft/azuredatastudio/assets/124254981/f8072efd-88e4-477e-97e6-dfa711ad3b08)

**Comment by corivera** [comment]
@kapilvaishna This issue has been fixed in the latest Windows Insiders build of ADS here: https://azuredatastudio-update.azurewebsites.net/latest/win32-x64-user/insider

Can you verify that the issue is resolved?

**Comment by kapilvaishna** [comment]
Issue is not repro in latest Windows Insiders build of ADS here: https://azuredatastudio-update.azurewebsites.net/latest/win32-x64-user/insider , So closing the bug.

NVDA fixed
https://github.com/microsoft/azuredatastudio/assets/124254981/69a810d1-5dbe-40e8-b143-1ccaecd9fff7

Jaws fixed
https://github.com/microsoft/azuredatastudio/assets/124254981/3a78ffe8-e0b8-4420-ac35-2b97a04b3d25
